go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/alicebob/miniredis/v2 v2.30.1
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/defsec v0.85.0
+	github.com/aquasecurity/defsec v0.85.1-0.20230414202454-d885472b0fd7
```

Comment:
note: Update to latest release when https://github.com/aquasecurity/defsec/pull/1280 is merged and defsec released.